dolphin/processor/config.py

```diff
@@ -81,7 +81,23 @@ def pixel_size(self):
         :return:
         :rtype:
         """
-        return self.settings['pixel_size']
+        if type(self.settings['pixel_size']) == float:
+            return [self.settings['pixel_size']]
+        else:
+            return self.settings['pixel_size']
+
+    @property
```

Comment:
I don't think `maximum_pixel_size` is very necessary to have as a class property. You can just use `np.max(self.pixel_size)` whenever you need this value. I suggest to remove this method altogether and to simply use `np.max(self.pixel_size)`.

Reply:
I don't think you're supposed to resolve a conversation until you have made the suggested changes, or until a resolution of the raised point has been achieved.

Reply:
Oops sorry, I though it was the previous issue, thanks for catching that